### PR TITLE
Fix Locations not showing on register.

### PR DESCRIPTION
**Pull request by Lentumunai-Mark**
**IMPORTANT: Where possible all PRs must be linked to a Github issue**

**Engineer Checklist**
- [ ] I have written **Unit tests** for any new feature(s) and edge cases for bug fixes
- [ ] I have added any strings visible on UI components to the `strings.xml` file
- [ ] I have updated the  [CHANGELOG.md](./CHANGELOG.md) file for any notable changes to the codebase
- [ ] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the project's style guide
- [ ] I have built and run the FHIRCore app to verify my change fixes the issue and/or does not break the app 
- [ ] I have checked that this PR does NOT introduce **breaking changes** that require an update to **_Content_** and/or **_Configs_**? _If it does add a sample here or a link to exactly what changes need to be made to the content._

**Code Reviewer Checklist**
- [ ] I have verified **Unit tests** have been written for any new feature(s) and edge cases
- [ ] I have verified any strings visible on UI components are in the `strings.xml` file
- [ ] I have verifed the [CHANGELOG.md](./CHANGELOG.md) file has any notable changes to the codebase
- [ ] I have verified the solution has been implemented in a configurable and generic way for reuseable components
- [ ] I have built and run the FHIRCore app to verify the change fixes the issue and/or does not break the app

### CODE REVIEW

**Comment by Lentumunai-Mark** [comment]
This PR entails 

Fixing the issue of locations not showing on the register

File: android/quest/src/main/java/org/smartregister/fhircore/quest/ui/multiselect/MultiSelectBottomSheetFragment.kt [hunk 1]
```diff
@@ -59,6 +59,7 @@ class MultiSelectBottomSheetFragment() : BottomSheetDialogFragment() {
       multiSelectViewModel.saveSelectedLocations(requireContext())
       appMainViewModel.run {
         if (requireContext().isDeviceOnline()) {
+          triggerOneTimeSync()
```
Comment (ellykits) [inline android/quest/src/main/java/org/smartregister/fhircore/quest/ui/multiselect/MultiSelectBottomSheetFragment.kt:62]:
```suggestion
         viewModelScope.launch { syncBroadcaster.runOneTimeSync() }
```

File: android/quest/src/main/java/org/smartregister/fhircore/quest/ui/multiselect/MultiSelectBottomSheetFragment.kt [hunk 2]
```diff
@@ -59,6 +59,7 @@ class MultiSelectBottomSheetFragment() : BottomSheetDialogFragment() {
...
           triggerSync()
```
Comment (ellykits) [inline android/quest/src/main/java/org/smartregister/fhircore/quest/ui/multiselect/MultiSelectBottomSheetFragment.kt:63]: Rename this to `schedulePeriodicSync`

**Review by ellykits — CHANGES_REQUESTED** [review]

File: android/quest/src/main/java/org/smartregister/fhircore/quest/ui/multiselect/MultiSelectBottomSheetFragment.kt (hunk 1 above)
Comment (Lentumunai-Mark) [reply android/quest/src/main/java/org/smartregister/fhircore/quest/ui/multiselect/MultiSelectBottomSheetFragment.kt:62]: @ellykits I created a function that addresses the suggested change [here](https://github.com/opensrp/fhircore/blob/a9cd5b56a6464045b1b56fa645c37bfb8f037ad4/android/quest/src/main/java/org/smartregister/fhircore/quest/ui/main/AppMainViewModel.kt#L319)

**Review by ellykits — APPROVED** [review]

**Pull request merged**